packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -380,6 +402,18 @@ export default function(
     const instance = new ctor(props, context);
     adoptClassInstance(workInProgress, instance);
 
+    const partialState = callGetDerivedStateFromProps(
+      workInProgress,
+      instance,
+      props,
+    );
+    if (partialState) {
```

Comment:
Maybe `!== null && !== undefined`? I think we wanted to avoid relying on truthiness to make engines' life easier.
It's okay if you're matching some existing code though.

Reply:
I initially mirrored [the check in `ReactFiberUpdateQueue`](https://github.com/facebook/react/blob/bd6b533c29cc617b4ddfc0a8b623e782c468cd52/packages/react-reconciler/src/ReactFiberUpdateQueue.js#L286), but I think it's worth changing here. 👍  Will do.